<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlsWidget</class>
 <widget class="QWidget" name="ControlsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
background: rgba(160,160,160,220);
}

QGroupBox{
background: rgba(160,160,160,220);
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>197</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background: transparent;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Illustration mode</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>2</number>
       </property>
       <item row="1" column="1">
        <widget class="QComboBox" name="statesType">
         <item>
          <property name="text">
           <string>Pixels</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2D plots</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phases</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="modeRasterRadioButton">
         <property name="text">
          <string>raster plots</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="modeEditRadioButton">
         <property name="text">
          <string>architecture</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="frType">
         <item>
          <property name="text">
           <string>Pixels</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2D plots</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="modeStatesRadioButton">
         <property name="text">
          <string>states</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="rOrientation">
         <item>
          <property name="text">
           <string>Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vertical</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="modeCRadioButton">
         <property name="text">
          <string>firing rates</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="rAxes_2">
         <property name="text">
          <string>Axes</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="rAxes">
         <property name="text">
          <string>Axes</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="netAnalysischeckBox">
         <property name="text">
          <string>Net analysis</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="networkMode">
         <item>
          <property name="text">
           <string>Mutual information</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Synchrony</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Transfer entropy</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="modeDefaultRadioButton">
         <property name="text">
          <string>Default</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSlider" name="speedSlider">
        <property name="minimumSize">
         <size>
          <width>38</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="speedLabel">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Speed: (30ms)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>show more</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="moreFrame">
   <property name="geometry">
    <rect>
     <x>466</x>
     <y>0</y>
     <width>189</width>
     <height>198</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="connectionsCheckBox">
          <property name="text">
           <string>edges</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="experimentCheckBox">
          <property name="text">
           <string>actions</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="updateSchemaCheckBox">
          <property name="text">
           <string>figure</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="detailsCheckBox">
          <property name="text">
           <string>details</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="updateGraphs">
          <property name="text">
           <string>graphs</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="learningCheckBox">
          <property name="text">
           <string>learning</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="firingRateLabel">
        <property name="text">
         <string>FR depth: 30</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="firingRateSlider">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>122</width>
          <height>29</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>199</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
